index.html

```diff
@@ -264,103 +264,96 @@ <h2>Intro Section of the Document</h2>
-          <dt>
-            <dfn id="form-stat">Form-Stat</dfn>
-          </dt>
-          <dd>
-            <p>Status: One of Initial, Current, Superseded or Obsolete</p>
-          </dd>
-          <dt>
-            <dfn id="form-summ">Form-Summ</dfn>
-          </dt>
-          <dd>
-            <p>Summary Document: Link to the summary document</p>
-          </dd>
-          <dt>
-            <dfn id="form-ver">Form-Ver</dfn>
-          </dt>
-          <dd>
-            <p>Version: A unique string for that entry&#39;s history that denotes the version of the entry that is linked. The version string SHOULD contain a UTC-based date in ISO 8601 format in the form of <code>YYYY-MM-DD</code>.</p>
-          </dd>
-        </dl>
+        <p>Each entry MUST contain the following information, and all parts of the entry MUST not conflict with existing bindings.</p>
+        <table class="def numbered"><!-- -->
+          <caption>
+            Entry format information
+          </caption>
+          <thead>
+            <tr>
+              <th>Information</th>
+              <th>Additional Notes</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr id="form-name">
+              <td>Name of the binding</td>
+              <td>Examples: <code>HTTP Binding</code>, <code>CoAP Binding</code></td>
+            </tr>
+            <tr id="form-link">
+              <td>Link to the binding document</td>
+              <td>Stable link whose content cannot change (e.g., a date, version number, etc.)  <br />
+                Examples: <code>https://www.w3.org/TR/wot/binding-templates/http-20240726/index.html</code>
+              </td>
+            </tr>
+            <tr id="form-pref">
+              <td>Binding ontology prefix</td>
+              <td>Examples: <code>htv</code>, <code>modv</code>, <code>cov</code></td>
+            </tr>
+            <tr id="form-id">
+              <td>Binding Identification in TD</td>
+              <td>URI Scheme or other TD terms reserved for this binding. <br />
+                Examples: <code>&quot;subprotocol&quot;:&quot;sse&quot;</code>, <code>&quot;href&quot;:&quot;http://example.com&quot;</code>, <code>&quot;contentType&quot;:&quot;application/json&quot;</code>
+                <ul>
+                  
+                  <li>In a TD, a binding SHOULD be identifiable by the terms in a form such as <code>href</code>, <code>contentType</code>, <code>subprotocol</code>, or connection information found in the root of the TD.<ul>
+                  <li>Reasons Behind the Requirement:<ul>
+                  <li>This avoids conflicts that are mentioned in the previous requirement</li>
+                  </ul>
+                  </li>
+                  <li>TODO: These terms should be refined based on the additions/changes to the TD 2.0 mechanism, e.g., introducing a <code>protocol</code> term or putting restrictions on URI scheme and <code>subprotocol</code> combination, data mapping, etc.</li>
```

Comment:
This can be an editor's note

Reply:
It is an editor's note already, right after the table. This points to commented code.
In https://github.com/w3c/wot-bindings-registry/pull/17/commits/beb5c446e9c8b333b3e93eee4de737475defc241 I removed the commented out TODO.